### DISCUSSION TITLE
请问有没有什么办法让每次使用抽卡180之后不保存当前的抽卡次数

### DISCUSSION THREAD

**Discussion by Mayoi3324** [Arondight/Adachi-BOT#841] [Q&A] [answered]
因为看起来用抽卡之后看起来后一次的次数是累加在前一次上的，会出现抽180次卡但是显示的五星次数加起来有200多次这样的，不太直观，想要改成不保存次数和保底然后增加抽卡次数的最大值
抽卡次数最大值似乎是改index.js？如果要改成不保存抽卡次数和当前保底状态，需要改哪里呢？

**Comment by Arondight** [comment, marked as answer]
https://github.com/Arondight/Adachi-BOT/blob/eab15fbefdcb2d35072ac7b8eca7f6d6b5e619c4/src/plugins/gacha/data.js#L279
